lib/rubygems/resolver/molinillo/lib/molinillo/resolver.rb

```diff
@@ -3,7 +3,7 @@
 module Gem::Resolver::Molinillo
   # This class encapsulates a dependency resolver.
   # The resolver is responsible for determining which set of dependencies to
-  # activate, with feedback from the the {#specification_provider}
```

Comment:
Please don't change vendored code directly. This will have to be changed upstream in Molinillo.